packages/terra-clinical-result/src/result-time-header-cell/ResultTimeHeaderCell.module.scss

```
@@ -0,0 +1,53 @@
+@import '../common/mixins';
+
+:local {
+  .clinical-result-time-header-cell {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+
+    .date {
+      @include terra-clinical-result-cell-truncate;
+      color: var(--terra-clinical-result-time-header-cell-date-color, rgba(28, 31, 33, 0.65));
+      flex: 0 1 auto;
+      font-size: var(--terra-clinical-result-time-header-cell-date-font-size, 0.92857em); // Must use ems for font scaling
+      font-weight: var(--terra-clinical-result-time-header-cell-date-font-weight, 400);
+      line-height: var(--terra-clinical-result-time-header-cell-date-line-height, 1.38461);
+      max-width: 100%;
+      min-height: var(--terra-clinical-result-time-header-cell-date-min-height, 1.38461em); // Set to same as line-height in ems
+      min-width: 10px; // Needed for IE10
+
+      &.hide-date {
+        @include terra-clinical-result-visually-hidden-text;
+        min-height: var(--terra-clinical-result-time-header-cell-date-min-height, 1.38461em); // Set to same as line-height in ems
+      }
+    }
+
+    .time {
+      @include terra-clinical-result-cell-truncate;
+      color: var(--terra-clinical-result-time-header-cell-time-color);
+      flex: 0 1 auto;
+      font-size: var(--terra-clinical-result-time-header-cell-time-font-size, 1em); // Must use ems for font scaling
+      font-weight: var(--terra-clinical-result-time-header-cell-time-font-weight, 400);
+      line-height: var(--terra-clinical-result-time-header-cell-time-line-height, 1.14285);
+      max-width: 100%;
+      min-height: var(--terra-clinical-result-time-header-cell-time-min-height, 1.14285em); // Set to same as line-height in ems
+      min-width: 10px; // Needed for IE10
+    }
+
+    /* stylelint-disable-next-line selector-max-compound-selectors */
+    .date + .time {
+      margin-top: var(--terra-clinical-result-time-header-cell-time-margin-top, 0.14285em); // Must use ems for font scaling
```

```suggestion
      margin-top: var(--terra-clinical-result-time-header-cell-margin-top, 0.14285em); // Must use ems for font scaling
```

Why does this change if both are present?